### PR TITLE
ping, ping6: add more information link

### PR DESCRIPTION
**Pull request by syleung**
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

Relates to issue https://github.com/tldr-pages/tldr/issues/6062

### CODE REVIEW

**Review by pixelcmtd — APPROVED** [review]

**Comment by syleung** [comment]
Thanks @pixelcmtd for your review in the other PR. I've also updated these links to use the official documentation.

File: pages/common/ping.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # ping
 
 > Send ICMP ECHO_REQUEST packets to network hosts.
+> More information: <https://man.openbsd.org/ping>.
```
Comment (marchersimon) [inline pages/common/ping.md:4]: Even tough `ping` originally comes from `BSD`, the version we currently use on Linux doesn't. So we should use a more general link
```suggestion
> More information: <https://manned.org/ping>.
```

File: pages/common/ping6.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
 # ping6
 
 > Send ICMP ECHO_REQUEST packets to network hosts via IPv6 address.
+> More information: <https://man.openbsd.org/ping6>.
```
Comment (marchersimon) [inline pages/common/ping6.md:4]:
```suggestion
> More information: <https://manned.org/ping6>.
```

File: pages/common/ping.md (hunk 1 above)
Comment (syleung) [reply pages/common/ping.md]: Thanks @marchersimon for the correction.

**Review by marchersimon — APPROVED** [review]
Thanks @syleung

**Pull request merged**